src/test/java/core/mate/academy/service/MachineServiceTest.java

```diff
@@ -14,7 +14,7 @@
 import java.util.Random;
 
 public class MachineServiceTest {
-    private static MachineService<Machine> machineService;
+    private static MachineService machineService;
```

Comment:
```suggestion
    private static MachineService<Machine> machineService;
```
you should not change tests